Dockerfile

```diff
@@ -1,59 +1,78 @@
-FROM node:22.14-bookworm-slim as builder
+# Stage: base image
+FROM node:22-alpine AS base
 
-ARG BUILD_NUMBER
-ARG GIT_REF
+LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
+
+RUN apk --update-cache upgrade --available \
+  && apk --no-cache add tzdata \
+  && rm -rf /var/cache/apk/*
+
+ENV TZ=Europe/London
+RUN ln -snf "/usr/share/zoneinfo/$TZ" /etc/localtime && echo "$TZ" > /etc/timezone
 
-RUN apt-get update && \
-    apt-get upgrade -y
+RUN addgroup --gid 2000 --system appgroup && \
+    adduser --uid 2000 --system appuser --ingroup appgroup
 
 WORKDIR /app
 
-COPY . .
+ARG BUILD_NUMBER
+ARG GIT_REF
+ARG GIT_BRANCH
 
-RUN CYPRESS_INSTALL_BINARY=0 npm ci --no-audit && \
-    npm run build && \
-    export BUILD_NUMBER=${BUILD_NUMBER} && \
-    export GIT_REF=${GIT_REF} && \
-    npm run record-build-info
+# Cache breaking and ensure required build / git args defined
+RUN test -n "$BUILD_NUMBER" || (echo "BUILD_NUMBER not set" && false)
+RUN test -n "$GIT_REF" || (echo "GIT_REF not set" && false)
+RUN test -n "$GIT_BRANCH" || (echo "GIT_BRANCH not set" && false)
 
-RUN npm prune --production
+# Define env variables for runtime health / info
+ENV BUILD_NUMBER=${BUILD_NUMBER}
+ENV GIT_REF=${GIT_REF}
+ENV GIT_BRANCH=${GIT_BRANCH}
 
-FROM node:22.14-bookworm-slim
-LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
+# Stage: build assets
+FROM base AS build
 
```

Comment:
I *think* that this wasn't working because `ARG GIT_REF` wasn't defined again in this second stage